pages/api/college-data.js

```diff
@@ -0,0 +1,48 @@
+// pages/api/college-data.js
+
+import fs from "fs/promises";
+import path from "path";
+
+export default async function handler(req, res) {
+  const { exam, counselling, category } = req.query;
+
+  const examMap = {
+    "JEE Main": "JEE",
+    "JEE Advanced": "JEE",
+    "MHT CET": "MHTCET",
+    KCET: "KCET",
+    NEET: "NEET",
+  };
+
+  const examFolder = examMap[exam] || "NEET";
+  const dataPath =
```

Comment:
my point is that this is such a weird code.. we can improve all this with a constant structure in db. no need to do all of this `public/data/ ` and individual logics. this is for next PR